src/components/ConfirmedRoute.tsx

```diff
@@ -34,11 +34,15 @@ type ConfirmedRouteProps = ConfirmedRoutePropsOnyxProps & {
     /** Whether it should have border radius */
     shouldHaveBorderRadius?: boolean;
 
+    /** Whether it should display the Mapbox map only when the route/coordinates exist otherwise
+     * it will display pending map icon */
+    shouldDisplayMapOnlyIfCoordinatesExist?: boolean;
+
```

Comment:
Do we need this to be a prop? Would you please describe the flow where we want to see the map without the coordinates? I'm having trouble thinking about when that would actually happen.

Reply:
Maybe `requireRouteToDisplayMap` is a better name? Or `displayPendingWithoutRoute`?

Reply:
>Do we need this to be a prop? Would you please describe the flow where we want to see the map without the coordinates? I'm having trouble thinking about when that would actually happen.

We display map without routes in confirmation page, for instance.

Reply:
Why? What would it look like to not do that? Don't we already have the route fetched every time we hit that page, or are there times where it's loading and would look bad without it?

Reply:
@neil-marcellini `ConfirmedRoute` is also used in money request creation flow (distance request) and in there, we displays the mapbox map even without the route, for instance, if the user hasn't yet selected waypoints of the distance request, that is what I meant.

Reply:
@neil-marcellini We use Confirmed Route in confirmation page too
https://github.com/Expensify/App/blob/b2887027c351e8f35b3603bfc43e439566f23964/src/components/MoneyRequestConfirmationList.tsx#L1159-L1160
If we want the same characteristics there too we can apply it to the confirmed route in general avoiding the prop. Let me know your thoughts.